compass/ocean/tests/drying_slope/namelist.init

```diff
@@ -0,0 +1,13 @@
+config_init_configuration='tidal_boundary'
+config_ocean_run_mode = 'init'
+config_tidal_boundary_vert_levels=10
+config_tidal_boundary_layer_type='sigma'
+config_tidal_start_dry=.true.
+config_tidal_boundary_left_bottom_depth=0.0
+config_tidal_boundary_use_distances=.true.
+config_tidal_forcing_monochromatic_baseline=10.0
+config_use_wetting_drying=.true.
+config_drying_min_cell_height=1.000001e-3
+config_use_tidal_forcing=.true.
+config_write_cull_cell_mask=.false.
+config_use_variable_drag=.true.
```

Comment:
`config_use_variable_drag=.false.` here too

Reply:
I just want to double check that you think we should skip these lines. I believe this would mean we would also remove `bottomDrag` as forcing input for the forward run. I think this would be a deviation from the legacy test case.
```
      if (config_use_variable_drag) then
        call mpas_pool_get_array(forcingPool, 'bottomDrag', bottomDrag)
        do iCell = 1, nCellsSolve
          bottomDrag(iCell) = config_tidal_forcing_left_Cd_or_n &
            + (yCell(iCell) - yMin) / (yMax - yMin)  * &
            (config_tidal_forcing_right_Cd_or_n - config_tidal_forcing_left_Cd_or_n)
        end do
      end if
```
https://github.com/E3SM-Project/E3SM/blob/4435bfa0e188f8419a4c2dc0dbcb88ceacea7e4f/components/mpas-ocean/src/mode_init/mpas_ocn_init_tidal_boundary.F#L220-L227

Reply:
That's true. I d true. I don't think that bottomDrag is usedon't think that bottomDrag is used in forward mode for this case, since it just uses Rayleigh drag (set via`config_Rayleigh_damping_coeff = ... `). So it should be fine to remove, right? If I recall correctly. I think the variable drag only used in the marsh test cases in legacy, which would require `config_use_variable_drag=.true.`

Reply:
Yes, you're right. I just double-checked the `ocn_vmix` routines. I'll remove `config_use_variable_drag` (it's false by default) and `bottomDrag` from the forcing stream.

Reply:
great, that sounds good to me

Reply:
@cbegeman, Is removing this option and the streams variable something we still would like to do?

Reply:
Thanks for reminding me. Let me test that out now.

Reply:
Looks good. No differences in the forward solution after the change.